compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/scene/UIViewComposeSceneLayer.uikit.kt

```diff
@@ -62,22 +62,26 @@ internal class UIViewComposeSceneLayer(
     private val backgroundView: UIView = object : UIView(
         frame = CGRectZero.readValue()
     ) {
-        init {
-            translatesAutoresizingMaskIntoConstraints = false
-            rootView.addSubview(this)
-            NSLayoutConstraint.activateConstraints(
-                getConstraintsToFillParent(this, rootView)
-            )
-        }
-
-        override fun pointInside(point: CValue<CGPoint>, withEvent: UIEvent?): Boolean {
-            //TODO pass invoke(true) on touch up event only when this touch event begins outside of layer bounds.
-            // Also it should be only one touch event (not multitouch with 2 and more touches).
-            // In other cases pass invoke(false)
-            onOutsidePointerEvent?.invoke(PointerEventType.Press)
-            onOutsidePointerEvent?.invoke(PointerEventType.Release)
-            return focusable
-        }
+        private var previousSuccessHitTestTimestamp: Double? = null
+        override fun hitTest(point: CValue<CGPoint>, withEvent: UIEvent?): UIView? =
+            if (
+                mediator.hitTestInteractionView(point, withEvent) == null &&
+                super.hitTest(point, withEvent) == this
+            ) {
+                if (previousSuccessHitTestTimestamp != withEvent?.timestamp) {
+                    // It is workaround to call onOutsidePointerEvent just once for one touch event.
+                    previousSuccessHitTestTimestamp = withEvent?.timestamp
+                    onOutsidePointerEvent?.invoke(PointerEventType.Press)
+                    onOutsidePointerEvent?.invoke(PointerEventType.Release)
```

Comment:
Please move this call into `touchEnded` like we discussed. It's not required to call if it's not focusable

Reply:
`PointerEventType.Release` moved to `touchesEnded`